contracts/src/tests/test_multisig.cairo

```diff
@@ -280,3 +281,25 @@ fn test_execute_confirmation_below_threshold() {
     Multisig::confirm_transaction(nonce: 0);
     Multisig::execute_transaction(nonce: 0);
 }
+
+#[test]
+#[available_gas(2000000)]
+#[should_panic(expected = ('only multisig allowed', ))]
+fn test_upgrade_not_multisig() {
+    let account = contract_address_const::<777>();
+    set_caller_address(account);
+
+    Multisig::upgrade(class_hash_const::<1>())
+}
+
+// wait until replace_class_syscall is implemented by cairo test
```

Comment:
Does Devnet support `replace_class_syscall`?
Can we add a couple of dummy integration contract upgrade tests?

Reply:
I think it should. I had some difficulty getting cairo-1 tests to run on starknet-devnet though but will try again today

Reply:
I'll wait until Alex's PR https://github.com/smartcontractkit/chainlink-starknet/pull/240/files is merged to add these contract upgrade tests because he has logic to get cairo-1 contracts deployed and working on devnet.